main.c

```diff
@@ -0,0 +1,63 @@
+#include "include/testsuite.h"
+
+int main()
+{
+	test_main_aes();
```

Comment:
Please print something before and after each test to let users know how far the tests got.

```suggestion
    printf("[1/56] Testing AES encryption...\n");
    rc = test_main_aes();
    printf("AES testing result: ");
    if (!rc) printf("PASSED\n"); else printf("FAILED\n");
```

Reply:
I just saw that your mains return `void`, so you can't check if a test passed or not this way. The idea to print the current test remains anyway.